src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AvatarImage.xml

```diff
@@ -37,7 +37,7 @@
                     <values>
                         <value name="groups">
                             <values>
-                                <value>admin:avatar_image:read</value>
+                                <value>admin:admin_user:avatar_image:read</value>
```

Comment:
Is it necessary since `avatar_image` is available as individual resource? (But I do feel as well, that `avatar_image` is not enough)

Scheme `{resource}:{resource}` sounds like something about sub-resource.

Though I don't have particular experience or opinion about this.

Reply:
[Link](https://github.com/Sylius/Sylius/pull/15441#discussion_r1367559743)

Reply:
Ok, currently, we use a serialization group for resources, even if it's a subresource. So I have left it as it was.